test/functional/datastoresrp/resources/mongodb_test.go

```diff
@@ -118,62 +118,6 @@ func Test_MongoDB_Recipe(t *testing.T) {
 	test.Test(t)
 }
 
-// Test_MongoDB_Recipe_Parameters validates the creation of a mongoDB from recipe with parameters passed by operator while linking recipe
-// and developer while creating the mongoDatabase resource.
-// If the same parameters are set by the developer and the operator then the developer parameters are applied in to resolve conflicts.
-// Container uses the mongoDB resource to connect to the mongoDB resource
-func Test_MongoDB_RecipeParameters(t *testing.T) {
-	t.Skip("Skipping test as creating/deleting cosmosdb resource is unreliable - https://github.com/radius-project/radius/issues/5929")
-
-	template := "testdata/datastoresrp-resources-mongodb-recipe-parameters.bicep"
```

Comment:
Is this file still around for us to clean up?

Reply:
Ah yes, delete this and the recipe file.